lib/Migration/Version002003000Date20220713094721.php

```diff
@@ -38,8 +38,8 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 		/** @var ISchemaWrapper $schema */
 		$schema = $schemaClosure();
 
-		if (!$schema->hasTable('recognize_files_images')) {
-			$table = $schema->createTable('recognize_files_images');
+		if (!$schema->hasTable('recognize_queue_imagenet')) {
```

Comment:
Should there be a cleanup for the old tables?

Reply:
The old tables were never released